docs/guidelines.md

```diff
@@ -552,7 +552,7 @@ include the `ngeo-layertree` directive.
  * @return {angular.Directive} The directive Definition Object.
  * @ngInject
  */
-gmf.layertreeComponent = function() {
```

Comment:
Oh wow, I did not have the doc directory on my radar. Nice catch!

Reply:
I think that's not really useful, because the doc is probably outdated. But Anyway, It's better like this.